hubble/executor/hubapi.py

```diff
@@ -250,3 +251,16 @@ def exist_local(uuid: str, tag: str = None) -> bool:
         return True
     except FileNotFoundError:
         return False
+
+
+def load_manifest(yaml_path: Path) -> Dict:
+    """Load manifest of executor from YAML file."""
+    with open(yaml_path / 'manifest.yml') as fp:
```

Comment:
`manifest.yml` has been merged to `config.yml` https://github.com/jina-ai/jina/pull/5101.

Please try to get infos from `config.yml` first then maybe try to read `manifest.yml` as a fallback.